modules/swagger-codegen/src/main/resources/nancyfx/api.mustache

```diff
@@ -6,6 +6,7 @@ using Sharpility.Base;
 using {{packageName}}.{{packageContext}}.Models;
 using {{packageName}}.{{packageContext}}.Utils;
 using NodaTime;
+{{#asyncServer}}using System.Threading.Tasks;{{/asyncServer}}
```

Comment:
You may to avoid a new line by using the following instead:
```
{{#asyncServer}}
using System.Threading.Tasks;
{{/asyncServer}}
```

Reply:
done